pkg/apis/extensions/types.go

```diff
@@ -87,6 +88,18 @@ type CPUTargetUtilization struct {
 	TargetPercentage int `json:"targetPercentage"`
 }
 
+// Alpha-level support for Custom Metrics in HPA (as annotations).
+type CustomMetricUtilization struct {
```

Comment:
Why is it called CustomMetricUtilization instead of CustomMetric?  It might be a value like QPS which is not a utilization?